themes/classic/templates/checkout/order-confirmation.tpl

```diff
@@ -92,16 +92,15 @@
     {/if}
   {/block}
 
-  {block name='customer_registration_form'}
-    {if $customer.is_guest}
-      <div id="registration-form" class="card">
+  {if !$registered_customer_exists}
+    {block name='conversion_form'}
+      <div class="card">
         <div class="card-block">
-          <h4 class="h4">{l s='Save time on your next order, sign up now' d='Shop.Theme.Checkout'}</h4>
-          {render file='customer/_partials/customer-form.tpl' ui=$register_form}
+          {include file='customer/_partials/conversion-form.tpl'}
```

Comment:
Naming is a bit awkward for me, as conversion has a completely different meaning most of the time. What do you think about `account-transformation-form.tpl`?